README.md

```diff
@@ -10,4 +11,31 @@ The project will consist of developing an open-source API that allows you to pas
 - Octokit.JS
 - Jest
 
+## Dev Runbook
+Dependancies: NodeJS, npm, GithubAPIKey 
+
+ 1. Pull the repository to your local setup
+ 2. Run `npm i`
+ 3. Make a `.env` file in the directory and add the variables: 
+	 **PORT**= **(port number you want to host the api)**
+	 **GITHUB_API_KEY**= **(Github API key)**
+4.  Run `npm run build:watch` on one terminal.
+5.  On another terminal, run `npm run start` to start the server on the port specified on. 
+6.  Now the nodejs server should be running! To test it just go to `localhost:{{PORT}}` and the text `TypeScript With Express` should be returned.
+
+## Setting up ElasticSearch Runbook (Dev/Local)
+-**docker pull docker.elastic.co/elasticsearch/elasticsearch:8.8.2** 
```

Comment:
Create a docker compose file or a bash script with that does all this setup

Reply:
Will do added to backlog.